image-viewer.py

```diff
@@ -1,37 +1,38 @@
-from tkinter import Tk, Label, Button
+from tkinter import Tk, Label, Button, Menu, filedialog
 from PIL import Image, ImageTk
 from os import listdir
 from os.path import isfile
 
 class ImageViewer():
     def __init__(self):
+        
+        default_directory = "./images"             
         self.current_image_index = 0
         self.image_files = []
-
-        for item in listdir("./images"):
-            if isfile(f"./images/{item}"):
-                self.image_files.append(item)
+        self.blank_img = Image.new('RGBA',(500,300),(0,0,0,0))
+        self.img_dir = default_directory
 
         self.root = Tk()
-        self.name_label = Label(self.root, text=self.image_files[self.current_image_index], bg="#1D1D1D", fg="#FFFFFF", font=("Arial 18"))
-        self.current_image = ImageTk.PhotoImage(Image.open(f"./images/{self.image_files[self.current_image_index]}"))
+        self.name_label = Label(self.root, text="No image found", bg="#1D1D1D", fg="#FFFFFF", font=("Arial 18"))
+        self.current_image = ImageTk.PhotoImage(self.blank_img)
         self.display = Label(image=self.current_image, width=500, height=300, bg="#1D1D1D")
         self.move_left_button = Button(self.root, text="<", font=("Calibri 20"), bg="#1C1C1C", fg="#FFFFFF", command=self.move_left)
         self.move_right_button = Button(self.root, text=">", font=("Calibri 20"), bg="#1C1C1C", fg="#FFFFFF", command=self.move_right)
+                
+
+        self.create_menu()
+        self.refresh()
+        
 
     def move_left(self):
         if self.current_image_index != 0:
             self.current_image_index -= 1
-            self.name_label.configure(text=self.image_files[self.current_image_index])
-            self.current_image = ImageTk.PhotoImage(Image.open(f"./images/{self.image_files[self.current_image_index]}"))
-            self.display.configure(image=self.current_image)
+            self.display_image()
 
     def move_right(self):
         if self.current_image_index != len(self.image_files) - 1:
```

Comment:
`!= len(self.image_files) - 1` <- not sure what this is preventing? Might not be needed.